static/app/components/replays/contextIcon.tsx

```diff
@@ -16,18 +17,23 @@ type Props = {
   showVersion?: boolean;
 };
 
-const LazyContextIcon = lazy(
-  () => import('sentry/components/events/contexts/contextIcon')
-);
+async function loadPlatformIcon() {
```

Comment:
`import('platformicons')` has type 
![Screenshot 2024-07-15 at 10 01 35 AM](https://github.com/user-attachments/assets/80b9886c-0fc2-440c-a812-b742e76c434a)
so, I had to try this